crates/autopilot/src/database/orders.rs

```diff
@@ -58,9 +46,27 @@ impl Postgres {
             &mut ex,
             &database::byte_array::ByteArray(order_uid.0),
             &database::orders::FeeUpdate {
-                surplus_fee: u256_to_big_decimal(&update.surplus_fee),
+                surplus_fee: u256_to_big_decimal(&quote.fee_amount),
                 surplus_fee_timestamp: Utc::now(),
-                full_fee_amount: u256_to_big_decimal(&update.full_fee_amount),
+                full_fee_amount: u256_to_big_decimal(&quote.full_fee_amount),
+            },
+        )
+        .await?;
+
+        database::quotes::save(
```

Comment:
I don't think we want to save the quote. I think this is the correct database method to use:
https://github.com/cowprotocol/services/blob/41ded83e30c8bc98e0e96566099c53363eb99791/crates/database/src/orders.rs#L273

Reply:
🙈 

Do you think we should rename the table to make things clearer?